react/features/conference/components/Indicators.native.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import { connect } from 'react-redux';
+
+import { JitsiRecordingConstants } from '../../base/lib-jitsi-meet';
+import {
+    isNarrowAspectRatio,
+    makeAspectRatioAware
+} from '../../base/responsive-ui';
+import { RecordingLabel } from '../../recording';
+import { VideoQualityLabel } from '../../video-quality';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * The indicator which determines whether the filmstrip is visible.
+     */
+    _filmstripVisible: boolean
+};
+
+/**
+ * A container that renders the conference indicators, if any.
+ */
+class Indicators extends Component<Props> {
+    /**
+     * Implements React {@code Component}'s render.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const _wide = !isNarrowAspectRatio(this);
+        const { _filmstripVisible } = this.props;
+
+        return (
+            <View
+                style = { [
+                    styles.indicatorContainer,
+                    _wide && _filmstripVisible && styles.indicatorContainerWide
+                ] }>
+                <RecordingLabel
+                    mode = { JitsiRecordingConstants.mode.FILE } />
+                <RecordingLabel
+                    mode = { JitsiRecordingConstants.mode.STREAM } />
+                <VideoQualityLabel />
+            </View>
+        );
+    }
+
+}
+
+/**
+ * Maps (parts of) the redux state to the associated {@code Indicators}'s props.
+ *
+ * @param {Object} state - The redux state.
+ * @private
+ * @returns {{
+ *     _filmstripVisible: boolean
+ * }}
+ */
+function _mapStateToProps(state) {
+    const { length: participantCount } = state['features/base/participants'];
+    const { visible } = state['features/filmstrip'];
+
+    return {
+        /**
+         * The indicator which determines whether the filmstrip is visible.
+         *
+         * @private
+         * @type {boolean}
+         */
+        _filmstripVisible: visible && participantCount > 1
```

Comment:
why the check for participantcount? also, please use the new helper utility for gettting the participants count, it's more future-proof.

Reply:
Sure, I change the function, thanks! But we need to check the participant counts, otherwise it won't properly work (see Filmstrip code)

Reply:
Checked that function and I'm afraid that won't work for us here. That function excludes the bot participants, that the Filmstrip doesn't and we need to behave exactly like the Filmstrip because thats position is the thing that what we want to address here.